File: HackerRank/Challenges/the-love-letter-mystery.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <string>
using namespace std;


int main() {
    int n, i, j, v;
    string s;
    cin >> n;
    while(n--){
        cin >> s;
        j = s.length() - 1;
        v = i = 0;
        while(j>i){
            v += abs((int)(s[j]-s[i]));
            j--;
            i++;
        }
        cout << v << endl;
    }
    return EXIT_SUCCESS;
}

File: HackerRank/Challenges/the-love-letter-mystery.py
n = int(input())
while(n != 0):
    s = input()
    v = i = 0
    j = len(s)-1
    while(j > i):
        v += abs(ord(s[j])-ord(s[i]))
        i+=1
        j-=1
    print(v)
    n-=1